package.json

```diff
@@ -261,10 +260,10 @@
     "typescript": "3.7.5"
   },
   "dependencies": {
-    "metals-languageclient": "0.1.14",
+    "coc.nvim": "0.0.74",
```

Comment:
`coc.nvim` is a runtime dependency, so I've moved it here (this is irrelevant since we usually do `yarn install` then build, but if you happened to do `yarn install --prod` instead then it would be a problem)